agent/api/ecsclient/client.go

```diff
@@ -288,13 +288,13 @@ func (client *APIECSClient) getCustomAttributes() []*ecs.Attribute {
 
 func (client *APIECSClient) SubmitTaskStateChange(change api.TaskStateChange) error {
 	// Submit attachment state change
-	if change.Attachments != nil {
+	if change.Attachment != nil {
```

Comment:
yeah, it's coming from https://github.com/aws/amazon-ecs-agent/blob/enis/agent/eni/watcher/watcher_linux.go#L141